shell/models/management.cattle.io.project.js

```diff
@@ -69,9 +69,13 @@ export default class Project extends HybridModel {
     return this.listLocation;
   }
 
-  async save() {
+  async save(forceReplaceOnReq) {
     const norman = await this.norman;
 
+    if (forceReplaceOnReq) {
+      norman.setReplaceFlagForNormanReq();
```

Comment:
Haven't tested yet, but you can remove the need for this by just passing in `norman.save({ replace: forceReplaceOnReq)` and then delete if from the `opt` as per the object.